pems-server/src/main/java/org/maroubra/pemsserver/monitoring/AbstractSensor.java

```diff
@@ -0,0 +1,10 @@
+package org.maroubra.pemsserver.monitoring;
+
+import io.reactivex.Flowable;
+
+public abstract class AbstractSensor {
```

Comment:
I think this was added by mistake as we changed the class to Sensor, can this be removed?